lib/transmission_api/client.rb

```diff
@@ -54,15 +54,31 @@ def find(id)
     response["arguments"]["torrents"].first
   end
 
-  def create(filename)
+  def create(filename, download_dir = nil, bandwidthPriority = nil, paused = nil)
```

Comment:
Ruby uses snake_case for variables, so we should use that on the arguments too

Reply:
I left those as close as possible to the ones on the API for lack of any better direction (the RPC api mixes things a lot...)